src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -244,7 +355,7 @@ int CallMain(CALL_APPLICATION_MAIN_DATA* data)
         fprintf(stderr, "Failed to initialize runtime: 0x%08x\n", result);
         return 1;
     }
-
+    
```

Comment:
Woah... did you just comment on your own PR?

![](http://img.pandawhale.com/52064-Inception-spinning-top-gif-rjRh.gif)